checkov/main.py

```diff
@@ -75,6 +76,8 @@
 logger = logging.getLogger(__name__)
 checkov_runners = [value for attr, value in CheckType.__dict__.items() if not attr.startswith("__")]
 
+RUN_NEW_SCA_PACKAGE_SCAN = os.getenv('RUN_NEW_SCA_PACKAGE_SCAN', '').lower() == 'true'
```

Comment:
the name "New" is not good, today it's new, tomorrow there is a newer.
maybe "RUN_SCA_PACKAGE_SCAN_V2"